src_js/components/sidebar/getSitemapUrls.ts

```diff
@@ -42,5 +42,20 @@ export default function getSitemapUrls(
   const [rootPage] = siteUrls.splice(rootIndex, 1);
   rootPage.title = Config.SITEMAP_SITE_TITLE;
 
+  siteUrls.sort((lhs, rhs) => {
+    const lhsOrder = lhs.sitemapOrder || 0;
+    const rhsOrder = rhs.sitemapOrder || 0;
+    if (lhs.external) {
+      // If lhs is external, it should come after all internal links.
+      return 1;
+    } else if (rhs.external) {
+      // If rhs is external but not lhs, then rhs should come after lhs.
+      return -1;
+    }
```

Comment:
Nice! Though I wonder whether this reverses the order of the `externalLink` entries. 

Do you think this condition should first check if _both_ `lhs` and `rhs` are external links? (I'm that case, we should return -1 to preserve the order.)

Reply:
Hmm, I think this might be dependent on the sorting algo used by the browser.

In reality, I think we should instead return `0` if `lhs.external && rhs.external`. I can push a commit for this real quick.

Reply:
Ah, good call!